native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/serializer/GlueSchemaRegistryKafkaSerializer.cs

```diff
@@ -0,0 +1,44 @@
+
+using System;
+using AWSGsrSerDe.common;
+
+namespace AWSGsrSerDe.serializer
+{
+    public class GlueSchemaRegistryKafkaSerializer
+    {
+        private readonly IDataFormatSerializer _serializer;
+        private readonly string _dataFormat;
+        private readonly Func<string, object, string> _schemaNamingStrategy;
+
+
+        public GlueSchemaRegistryKafkaSerializer()
+        {
+            // TODO: remove hardcode property with Config instead
+            _dataFormat = GlueSchemaRegistryConstants.DataFormat.AVRO.ToString();
+            _schemaNamingStrategy = (topic, data) => topic;
+            
+            _serializer = SerializerFactory.GetInstance().GetSerializer(_dataFormat);
```

Comment:
Can this be dynamic per `Serialize` method invocation?

Reply:
sure.